dashboards/mongodb-prometheus/mongodb.json

```diff
@@ -0,0 +1,1714 @@
+                "query": "FROM Metric WITH mongodb_rs_members_health['latest'] AS mh SELECT latest(CASES(mh=1 AS 'Up', mh=0 as 'Down')) AS 'Status', latest(mongodb_members_lastHeartbeatRecv) AS 'Latest Heartbeat', latest(mongodb_members_optimeDate) AS 'Latest Oplog Entry' WHERE mongodb_cluster_name IN ({{select_instance}}) AND member_idx IN ({{select_member}}) AND member_state IN ({{select_member_state}}) AND rs_nm IN ({{select_replicaset}}) FACET mongodb_cluster_name AS 'Cluster', rs_nm AS 'ReplicaSet' ,  member_idx AS 'Member', member_state AS 'Member State'"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            }
+          }
+        },
+        {
+          "title": "Ping Times",
+          "layout": {
+            "column": 1,
+            "row": 12,
+            "width": 5,
+            "height": 3
+          },
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT latest(mongodb_rs_members_pingMs) FROM Metric WHERE metricName='mongodb_rs_members_pingMs' AND mongodb_cluster_name IN ({{select_instance}}) AND member_idx IN ({{select_member}}) AND member_state IN ({{select_member_state}}) AND rs_nm IN ({{select_replicaset}}) FACET member_idx TIMESERIES AUTO"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "units": {
+              "unit": "MS"
+            },
+            "yAxisLeft": {
+              "zero": true
+            }
+          }
+        },
+        {
+          "title": "Average replication lag",
+          "layout": {
+            "column": 6,
+            "row": 12,
+            "width": 7,
+            "height": 3
+          },
+          "visualization": {
+            "id": "viz.area"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT (latest(mongodb_members_lastHeartbeat) - latest(mongodb_members_optimeDate)) / 1000 WHERE member_state='SECONDARY' AND mongodb_cluster_name IN ({{select_instance}}) AND member_idx IN ({{select_member}}) AND rs_nm IN ({{select_replicaset}}) FACET rs_nm, member_idx TIMESERIES AUTO"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "units": {
+              "unit": "MS"
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "variables": [
```

Comment:
in the variables section the syntax is slightly different:
```
  "variables": [
    {
      "name": "hostname",
      "items": null,
      "defaultValues": [
        {
          "value": {
            "string": "*"
          }
        }
      ],
      "nrqlQuery": {
        "accountIds": [],
        "query": "FROM Metric select uniques(hostname)where metricName = 'windows_service_state' since 7 days ago LIMIT MAX"
      },
```

Reply:
it is `"accountIds": [],`

Reply:
👍